nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -743,6 +755,12 @@ public void setUpClient(final ProcessContext context) throws TlsException {
             okHttpClientBuilder.cache(new Cache(getETagCacheDir(), maxCacheSizeBytes));
         }
 
+        if(context.getProperty(SUPPORT_HTTP2_PROTOCOL).asBoolean()) {
```

Comment:
Should there be a space after the `if`?
```suggestion
        if (context.getProperty(SUPPORT_HTTP2_PROTOCOL).asBoolean()) {
```